### PR TITLE
Add DMS template

### PR DESCRIPTION
**Pull request by mgblythe**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

*
*
*

### CODE REVIEW

**Comment by mgblythe** [comment]
@bmoore-msft -  I think this one is ready to go.

CC: @tfitzmac, @MashaMSFT

File: 101-azure-database-migration-simple-deploy/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            "location": "[parameters('location')]",
+            "properties": {
+                "virtualSubnetId": "[resourceId('Microsoft.Network/virtualNetworks/subnets', parameters('vnetName'), parameters('subnetName'))]"
+            },
+            "sku": {
+                "tier": "Standard",
+                "size": "1 vCores",
+                "name": "Standard_1vCores"
+            },
+            "dependsOn": [
```
Comment (bmoore-msft) [inline 101-azure-database-migration-simple-deploy/azuredeploy.json:71]: move this above properties... see: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md#sort-order-of-properties

**Comment by mgblythe** [comment]
@bmoore-msft - fixed the ordering issue, and yes this is a doc sample. Not sure what you and @tfitzmac have decided on re: marking as docs-related, but happy to add whatever info is required.

**Comment by bmoore-msft** [comment]
add the "docOwner" property to the metadata.json file - and add your gh userId as the property value (assuming you're the docOwner ;))

**Comment by bmoore-msft** [comment]
@MashaMSFT - check this PR for updates that may be needed to documentation that references this sample.

**Comment by mgblythe** [comment]
@bmoore-msft - I'm adding this for @MashaMSFT so her docs will align with this. Let me know if you need anything else to merge.

**Comment by MashaMSFT** [comment]
@bmoore-msft looks good to me!

**Pull request merged**